pkg/storage/bloom/v1/bloom_tokenizer.go

```diff
@@ -63,9 +74,43 @@ func (bt *BloomTokenizer) GetNGramSkip() uint64 {
 	return uint64(bt.lineTokenizer.Skip)
 }
 
-// TODO: Something real here with metrics
-func newMetrics(_ prometheus.Registerer) *metrics {
-	return &metrics{}
+func newMetrics(r prometheus.Registerer, namespace, subsystem string) *metrics {
+	return &metrics{
+		sbfCreationTime: promauto.With(r).NewCounter(prometheus.CounterOpts{
+			Name:      "bloom_creation_time",
+			Help:      "Time spent creating sbfs",
```

Comment:
could you please replace sbfs here in the description with the full form of this term?